bin/sign.py

```diff
@@ -198,7 +201,7 @@ def auth(tenant_id: str, client_id: str) -> JSON:
             "SubjectName": f"CN={client_id}.microsoft.com",
             "StoreLocation": "LocalMachine",
             "StoreName": "My",
-            "SendX5c" :  "true"
+            "SendX5c": "true",

```

Comment:
No change except whitespaces. Is this formatting fix intentional?

Reply:
Yes. This change is introduced after formatting with Black